src/shared/utils/convertUserInput.test.ts

```diff
@@ -67,3 +70,67 @@ describe('convertUserInputToNumOrNaN', () => {
     expect(convertUserInputToNumOrNaN(event)).toEqual(0)
   })
 })
+
+describe('convertUserInputValueToNumOrNaN', () => {
+  it('should convert undefined to NaN', () => {
+    let result = convertUserInputValueToNumOrNaN()
+    expect(typeof result === 'number')
+    expect(result !== result)
+
```

Comment:
i think you should add an expect for `isNan` like 
```ts
expect(Number.isNan(result)).toBeTruthy()
```
to these tests

(i also prefer `toBeTruthy` because it's more explicit)

Reply:
oh yeah good call. Done.

Reply:
And... actually, `expect(result !== result)` doesn't assert anything. It always passes even if the expression inside is false 😅 

So this change was definitely needed, just to get the tests to run correctly.